packages/venia-concept/src/index.js

```diff
@@ -37,11 +36,9 @@ ReactDOM.render(
         apollo={{ link: authLink.concat(Adapter.apolloLink(apiBase)) }}
         store={store}
     >
-        <WindowSizeContextProvider>
-            <ToastContextProvider>
-                <App />
-            </ToastContextProvider>
-        </WindowSizeContextProvider>
+        <Peregrine>
```

Comment:
How set are we on this name? I feel like this should be `PeregrineContextProvider`. Also, I know that you added `defaultProps` for toasts and window_size but I feel like we should just be explicit here. The less magic the better since people will be referencing this in the future and should be able to, at a glance, understand what exactly the top-level is doing.

Reply:
Naming is flexible. I'm not sure if there was an intention here that Peregrine might wrap more than just ContentProviders at some point, but if Providers is its only responsibility, this name change makes sense to me.

My concern with an explicit config prop is the reason we're creating this component now; I anticipate if we proceed with replacement of Redux with Peregrine hooks, our list of providers is going to be very large. I would be fine removing the defaultProp if we had a flag to use all (which is now just equivalent to not passing the prop).

Reply:
The only thing a root-level `Peregrine` component would ever do is provide context, and it enables all the features of the package, so I figure `Peregrine` is adequate here. But I don't mind renaming it; exports can always be renamed by the importer.

Reply:
I'd like to change the export/component at least. We can import as `Peregrine` here.